packages/terser/src/index.js

```diff
@@ -147,3 +150,49 @@ function spawn(cmd, args) {
     });
   })
 }
+
+function main() {
```

Comment:
please don't do unrelated refactorings. this just makes me search around to see what you changed.

Especially as you're on your first few contributions I would like to get minimal deltas where every edit is related to making the new test pass please

Reply:
I needed to do this refactor so that this file wouldn't jsut execute the CLI work flow when required into the tests